weatherman.py

```diff
@@ -0,0 +1,212 @@
+from argparse import ArgumentParser
+from collections import namedtuple
+from termcolor import colored
+
+
+import glob
+import calendar
+
+Date = namedtuple('Date', ['year', 'month', 'day'])
+
+
+class Temperature:
+    def __init__(self, date, max_temp, mean_temp, min_temp, dew_point,
+                 mean_dew_point, min_dew_point, max_humidity, mean_humidity,
+                 min_humidity, max_sea_level_pressure, mean_sea_level_pressure,
+                 min_sea_level_pressure, max_visibility, mean_visibility,
+                 min_visibility, max_wind_speed, mean_wind_speed,
+                 max_gust_speed, percipitation, cloud_cover, events, wind_dir):
+
+        self.date = date
+        self.max_temp = max_temp
+        self.mean_temp = mean_temp
+        self.min_temp = min_temp
+        self.dew_point = dew_point
+        self.mean_dew_point = mean_dew_point
+        self.min_dew_point = min_dew_point
+        self.max_humidity = max_humidity
+        self.mean_humidity = mean_humidity
+        self.min_humidity = min_humidity
+        self.max_sea_level_pressure = max_sea_level_pressure
+        self.mean_sea_level_pressure = mean_sea_level_pressure
+        self.min_sea_level_pressure = min_sea_level_pressure
+        self.max_visibility = max_visibility
+        self.mean_visibility = mean_visibility
+        self.min_visibility = min_visibility
+        self.max_wind_speed = max_wind_speed
+        self.mean_wind_speed = mean_wind_speed
+        self.max_gust_speed = max_gust_speed
+        self.percipitation = percipitation
+        self.cloud_cover = cloud_cover
+        self.events = events
+        self.wind_dir = wind_dir
+
+
+def handle_year(year, file_path):
+    text_files = glob.glob1(file_path, '*.txt')
+    matched_files = [s for s in text_files if year in s]
```

Comment:
can you name indexing something else?
Like for file in text_files

Reply:
Done